Makefile

```diff
@@ -78,13 +78,13 @@ endif
 endif
 endif
 
-OPENRA_UTILITY = ENGINE_DIR=".." $(MONO) --debug bin/OpenRA.Utility.exe
+OPENRA_UTILITY = ENGINE_DIR=".." $(MONO) --debug bin/OpenRA.Utility.dll
 
 ##################### DEVELOPMENT BUILDS AND TESTS #####################
 #
 all:
 	@command -v $(firstword $(MSBUILD)) >/dev/null || (echo "OpenRA requires the '$(MSBUILD)' tool provided by Mono >= 5.18."; exit 1)
-	@$(MSBUILD) -t:Build -restore -p:Configuration=Release -p:TargetPlatform=$(TARGETPLATFORM)
+	@$(MSBUILD) -t:Build -restore -p:Configuration=Release -p:TargetPlatform=$(TARGETPLATFORM) -p:TargetFramework=netstandard2.1 -p:Mono=true -p:DefineConstants="MONO"
```

Comment:
Do we really need to be defining `TargetFramework` here? The csproj should already be doing the right thing based on the `Mono` variable.